python/embag.cc

```diff
@@ -89,9 +88,8 @@ PYBIND11_MODULE(libembag, m) {
             throw std::runtime_error("Can only iterate array RosValues");
         }
       }, py::keep_alive<0, 1>() /* Essential: keep object alive while iterator exists */)
-      .def("__getattr__", [](Embag::VectorItemPointer<Embag::RosValue> &v, const std::string &attr) {
```

Comment:
It almost looks like you're removing the `[]` operator here. Is this correct, or is that still supported by the `getitem` below?

Reply:
The `[]` is part of the syntax for defining a lambda. The actual `[]` operator in python is defined by `__getitem__`. The `__getattr__` interface (which provides `.` access) is still supported, it just doesn't have a lambda that wraps the `getField` and instead calls `getField` directly.